gst/nnstreamer/tensor_common.c

```diff
@@ -856,6 +856,35 @@ gst_tensors_config_from_structure (GstTensorsConfig * config,
   return TRUE;
 }
 
+/**
+ * @brief Parse caps from peer pad and set tensors config.
+ * @param pad GstPad to get the capabilities
+ * @param config tensors config structure to be filled
+ * @param is_fixed flag to be updated when peer caps is fixed (not mandatory, do nothing when the param is null)
+ * @return TRUE if peer has tensor caps
+ */
+gboolean
+gst_tensors_config_from_peer (GstPad * pad, GstTensorsConfig * config,
+    gboolean * is_fixed)
+{
+  GstCaps *peer_caps;
+  GstStructure *structure;
+  gboolean ret = FALSE;
+
```

Comment:
Let's check whether params are null or not.

Reply:
gst_pad_peer_query_caps() and gst_tensors_config_from_structure() 
these functions check the null param (pad and config), so unnecessary.